clients/pkg/logentry/stages/json.go

```diff
@@ -92,8 +93,28 @@ func parseJSONConfig(config interface{}) (*JSONConfig, error) {
 	return cfg, nil
 }
 
+func (j *jsonStage) Run(in chan Entry) chan Entry {
+	out := make(chan Entry)
+	go func() {
+		defer close(out)
+		for e := range in {
+			err := j.process(e.Extracted, &e.Line)
+			if err != nil && j.cfg.DropMalformed {
+				continue
+			}
+			out <- e
+		}
+	}()
+	return out
+}
+
 // Process implements Stage
-func (j *jsonStage) Process(labels model.LabelSet, extracted map[string]interface{}, t *time.Time, entry *string) {
+func (j *jsonStage) Process(_ model.LabelSet, extracted map[string]interface{}, _ *time.Time, entry *string) {
+	//nolint:errcheck
+	j.process(extracted, entry)
+}
```

Comment:
The function `Process()` is obsolete, because `jsonStage` now implements the `Stage` interface.